roslibrust_genmsg/test_package/src/test_main.cpp

```diff
@@ -14,5 +14,7 @@ int main() {
     geometry_msgs::Polygon polygon;
     polygon.points.push_back(point);
 
+    std::cout << "Polygon: " << polygon << std::endl;
```

Comment:
Just to confirm this is now an intentional part of the test to exercise the serialization portions?

Reply:
Where the test is: does compilation fail or not. There's no verification of the output at all.